2017-11-27T01:07:45.548Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'DELBCG01' as the object already exists)
    at /usr/local/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2017-11-27T01:07:45.550Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'DELBCG01' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'DELBCG01' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/usr/local/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:888:34)"}$
